### DISCUSSION TITLE
Why the mAP is almost 0 for the first 20 epochs?

### DISCUSSION THREAD

**Discussion by lettuan** [rwightman/efficientdet-pytorch#203] [General]
When I train the EfficientDet D0 using your code, I see that in about the first 20 epochs the mAP on validation set is almost zero. Why do you have a curve like this? Does this long warmup contribute to the final result?

**Comment by rwightman** [comment]
I suggest searching
